pkg/chrootarchive/diff_unix.go

```diff
@@ -49,6 +51,10 @@ func applyLayer() {
 		fatal(err)
 	}
 
+	if inUserns {
+		options.InUserNS = true
```

Comment:
Hmm, @mwhudson managed to trigger a panic here on `s390x`, so I wonder if there's some edge case where `json.Unmarshal` doesn't create the struct?